flycheck-coala.el

```diff
@@ -34,10 +34,10 @@
 (flycheck-define-checker coala
   "A checker using coala.
 See URL `https://coala.io'."
-  :command ("coala" "--format" "{line}:{column}:{severity_str}:{message}" "--find-config" "--files" source)
-  :error-patterns  ((error (or "None" line) ":" (or "None" column) ":MAJOR:" (message))
-                    (warning (or "None" line) ":" (or "None" column) ":NORMAL:" (message))
-                    (info (or "None" line) ":" (or "None" column) ":INFO:" (message)))
+  :command ("coala" "--format" "{line}:{column}:{severity_str}:{message}" "--find-config" "--limit-files" source-original)
```

Comment:
But won't `limit-files` filter *out* given files? Wouldn't this render the plugin completely useless?

Reply:
https://github.com/coala/coala-emacs/commit/04adb144cda16535359f3be9fa492eb52c0e7120 changed `--limit-files` to `--files`.  The `--limit-files` was put in https://github.com/coala/coala-emacs/commit/88315e627b4ff8f04629a312e4b453d939a65bf9. Also, appeal to authority: During the hackathon, this was suggested to me by @sils.